### DISCUSSION TITLE
error on textareaeditor

### DISCUSSION THREAD

**Discussion by eriksetiadi-pro** [serenity-is/Serenity#6052] [Help]
![image](https://user-images.githubusercontent.com/68863121/138423597-6548ca6b-1c01-4dec-ad29-958da3dcfde5.png)

i use textareaeditor and get this error when i want to get value of textareaeditor. how can solve this?

**Comment by VictorTomaili** [comment]
It seems your scripts are not built well. Try to rebuild it again and check your error output.
